### ISSUE TITLE
"Schedulable?" on wishlist may not be accurate

### ISSUE THREAD

**Issue by JKobyP** [hacsoc/the_jolly_advisor#206]
I'm seeing a number of classes as schedulable because they're offered in the current semester. I'm scheduling for next semester, so that's misleading.

Example (3/6):
EECS 444

**Comment by ajm188** [comment]
I agree that could be misleading. Do you have any ideas for what the "right" thing would be instead?

**Comment by StephHippo** [comment]
Schedulable should shift to the next semester after drop/add ends, if we can figure that out. 

Since drop/add is usually the first 2 weeks of the semester so maybe earliest date a class starts that semester + 2 weeks?

**Comment by ajm188** [comment]
Alternatively, we could parameterize `schedulable?` to take a semester, then we just list out whether the course is schedulable from every semester from the current date to the last semester that's in the db. Since that will likely only be a semester or two ahead of the current date, it won't actually be _that_ bad.

**Comment by StephHippo** [comment]
You're losing me here. Maybe add an example?

"we just list out whether the course is schedulable from every semester from the current date to the last semester that's in the db"

**Comment by ajm188** [comment]
So the page would look like (pretend 132 is not offered Spring 2016):

```
Course      |  Schedulable?
EECS 132    Spring 2016, Fall 2016, Fall 2017
```
